src/libvirtApi/domain.js

```diff
@@ -765,6 +765,39 @@ export function domainGetCapabilities({ connectionName, arch, model }) {
     return call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'GetDomainCapabilities', ['', arch, model, '', 0], { timeout, type: 'ssssu' });
 }
 
+export function domainGetStartTime({
+    connectionName,
+    vmName,
+}) {
+    return cockpit.user().then(loggedUser => {
+        const options = { err: "message" };
+        if (connectionName === "system")
+            options.superuser = "try";
+
+        /* The possible paths of logfiles path, as of libvirt 9.3.0 are defined in:
+         * https://gitlab.com/libvirt/libvirt/-/blob/v9.3.0/src/qemu/qemu_conf.c#L153
+         * There libvirt defines 3 possible directories where they can be located
+         * - /root/log/qemu/ - That is however only relevant for "qemu:///embed", and not cockpit's use case
+         * - /var/log/libvirt/qemu/ - For privileged (qemu:///system) VMs
+         * - ~/.cache/libvirt/qemu/logs - For non-privileged  (qemu:///session) VMs
+         */
+        const logFile = connectionName === "system" ? `/var/log/libvirt/qemu/${vmName}.log` : `${loggedUser.home}/.cache/libvirt/qemu/log/${vmName}.log`;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1069-20230626-124631-ca214402-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)